StylableSwiftUI/Classes/WithStyleIdentifier.swift

```diff
@@ -37,19 +37,24 @@ public struct WithStylistIdentifier<Content: View>: View {
 
     @Environment(\.currentStylableGroup) private var currentStylableGroup
 
-    private let tokens: [String]
+    private let tokens: [StylistIdentifier]
```

Comment:
This parameter doesn't make sense being called "token" anymore :)

Reply:
Done